docs/faq.md

```diff
@@ -32,19 +32,73 @@ See the Semgrep feature and pricing comparison at [r2c.dev/pricing](https://r2c.
 For the Semgrep Team edition, a user is any project contributor whose pull request or code was scanned in the billing cycle. For example, if a project has 4 unique contributors who pushed code during the period, and Semgrep runs on all their pull requests, there are 4 users. If these unique contributors push to many projects, they will only be counted once; please feel free to scan often and widely. Semgrep does not charge on a line-of-code or per-project basis. 
 
 # Comparisons
+### How is Semgrep different than $OTHER_TOOL or $GENERIC_[SAST](https://en.wikipedia.org/wiki/Static_application_security_testing)?
 
-#### How is Semgrep different than linters?
-Linters use static analysis but have a narrower scope for analysis (most rules typically operate on a single line).
+Semgrep is an open-source tool with a simple syntax for writing rules: if you can write code, you can write a Semgrep rule — no program analysis PhD required!
 
-Semgrep incorporates rulesets inspired by the rules of many popular linters and checkers, including ESLint, RuboCop, Bandit, and FindSecBugs. But Semgrep also allows you to enable multiple rulesets at the same time without adding linter-specific artifacts or installation to your code repository.
+To our knowledge, the only other tool with the explicit goal of allowing custom rules is Semmle’s proprietary tool, CodeQL. CodeQL has a domain-specific language which is extremely powerful but is designed for those with significant program analysis expertise, whereas Semgrep is designed for the security engineer* ** *or developer who wants to automate code review. Our goal is to make writing a Semgrep rule as easy as copying the code you want to find—and letting the Semgrep engine to make the rule and autofix high-quality enough to run in CI or the editor.
```

Comment:
```suggestion
To our knowledge, the only other tool with the explicit goal of allowing custom rules is GitHub’s proprietary tool, CodeQL. CodeQL has a domain-specific language which is extremely powerful but is designed for those with significant program analysis expertise, whereas Semgrep is designed for the security engineer* ** *or developer who wants to automate code review. Our goal is to make writing a Semgrep rule as easy as copying the code you want to find—and letting the Semgrep engine to make the rule and autofix high-quality enough to run in CI or the editor.
```

For above, it looks like GitHub isn’t really using the Semmle brand anymore, so I expect most people will already (or very soon) see CodeQL as a GitHub thing rather than a Semmle thing.

Separately, looks like there is a Markdown glitch after "…whereas Semgrep is designed for the security engineer” with some extra `*` characters. I’m not sure what you were trying to make bold there.

Reply:
Thanks for the * catch too -- fixed, was just markdown editor vestigal stuff